### PR TITLE
v1.16 / v1.17 | clearlinux: decertify a fake distribution

### PR DESCRIPTION
**Pull request by KyleSanderson**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [*] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [*] Did you include the product/project logo in SVG, EPS or AI format?  
* [*] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [*] If your product/project is open source, did you include the `repo_url`?
* [*] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

So, clearlinux very clearly lied about their eligibility. The distribution frequently blows up, and does not result in a functional k8s install.

https://github.com/clearlinux/distribution/issues/2109
https://github.com/clearlinux/cloud-native-setup/issues/319

Clear Linux has no purpose as being "certified" beyond a horrible marketing ploy (cc @CraigSterrett @taylorwaggoner ). The installation never worked, and still does not work today. This PR removes Intel Clear Linux as it's completely nonfunctional and Clearly does not work.

Basically, I feel as if the CNCF has clearly mislead me by granting this guarantee, I now have hundreds of these machines that are a single upgrade away from being bricked.

### CODE REVIEW

**Comment by dankohn** [comment]
I've reviewed the initial conformance submission and the bug reports that you link to and disagree with your summary that it "never worked" and "isn't working today". Instead, you seem to be hitting an unfortunate corner case in a regression.

I'd encourage you to either work with the Clear Linux team to resolve the regression or consider switching to a different Certified Kubernetes distribution.

Going forward, there is a process for revoking a certification described in https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md but it is based on conformance tests not being replicable, which is not the case here.

**Comment by KyleSanderson** [comment]
cc @dankohn https://github.com/clearlinux/cloud-native-setup/issues/319#issuecomment-702009337
